app/kudos/views.py

```diff
@@ -289,6 +289,22 @@ def send_2(request):
     if _id and not str(_id).isdigit():
         raise Http404
 
+    username = request.GET.get('username', None)
+    user = {}
+
+    if username:
+        profiles = Profile.objects.filter(handle__iexact=username)
+
+        if profiles.exists():
+            profile = profiles.first()
```

Comment:
If we had two users with handles "zoek1" and "zoek1asdfasdf", and I wanted to send a kudos to "zoek1asdfasdf", would this work?

Reply:
Sure, It works because the filter method uses iexact on the handle attribute, in the previous bounty you recommended that